examples/window-app/common/window-app.zap

```diff
@@ -18,13 +18,13 @@
   "package": [
     {
       "pathRelativity": "relativeToZap",
-      "path": "../../../src/app/zap-templates/zcl/zcl.json",
+      "path": "../../../../../../../../git/chip/main/src/app/zap-templates/zcl/zcl.json",
```

Comment:
There is an issue with those paths, here and Line 27, as they are relative to your setup.